src/roslyn/eng/Publishing.props

```diff
@@ -5,8 +5,9 @@
   </PropertyGroup>
 
   <!-- Update Artifacts with Kind=Package to have additional metadata item Category="ToolingPackage".
-       Depending on channel configuration, this means that these assets could be pushed to a different feed. -->
-  <ItemGroup>
+       Depending on channel configuration, this means that these assets could be pushed to a different feed.
+       Do not include these for source-only builds since these files aren't intended to be published in that mode. -->
```

Comment:
Just curious, why are those not intended to get published? Also this item group here only does an Update not an Include. Why is that a problem?